test/list.test.ts

```diff
@@ -1,6 +1,6 @@
 import {expect} from "chai";
-
 import {BitList, bitLength} from "../src/list";
+import {it} from "mocha";
```

Comment:
maybe we should import `describe` too?

Reply:
Actually I found out problem. If tsconfig includes only "src" (to avoid emitting test types), cli works because it uses babel register. That's why I've added tsconfig.build.json that includes only src and it's used only when building